drivers/hmis/lib/tasks/activity_logs.rake

```diff
@@ -0,0 +1,4 @@
+desc 'expose log processor job for cron-job'
+task process_activity_logs: [:environment] do
+  Hmis::ActivityLogProcessorJob.perform_now
```

Comment:
Oh, excellent that this is a DJ already, should be easy enough to move to `grda_warhouse:hourly`